packages/flutter/test/material/list_tile_test.dart

```diff
@@ -1127,4 +1137,51 @@ void main() {
     expect(tester.getRect(find.byType(Placeholder).at(0)), const Rect.fromLTWH(800.0 - 16.0 - 24.0,        16.0, 24.0, 56.0));
     expect(tester.getRect(find.byType(Placeholder).at(1)), const Rect.fromLTWH(800.0 - 16.0 - 24.0, 88.0 + 16.0, 24.0, 56.0));
   });
+  testWidgets("ListTile doesn't accept focus when disabled", (WidgetTester tester) async {
```

Comment:
Could the test title be more generalized to testing ListTile focus? The first half of the test does check for the ListTile accepting focus when enabled, unless this setup is necessary testing focus when disabled. (This question applies for all the disabled widget tests)

Reply:
Sure.  I changed it to "ListTile only accepts focus when enabled", since I think that covers both cases without being super long.